src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLanguageServerClient.cs

```diff
@@ -254,6 +263,9 @@ private async Task EnsureContainedLanguageServersInitializedAsync()
         }
 
         await Task.WhenAll(clientLoadTasks).ConfigureAwait(false);
+
+        // We only want to mark the server as activated after the delegated language servers have been initialized.
+        _lspServerActivationTracker.Activated();
```

Comment:
My preference would be to remove this entire method, as it was only introduced to work around this issue anyway, but doing so in this PR seems a little aggressive, lest this causes problems and needs to be reverted.

Reply:
Maybe worth filing a 17.9 task for